### ISSUE TITLE
Can't log in

### ISSUE THREAD

**Issue by R3Ked** [thefifthmatt/windmill-client#41]
I'm trying to log in with my google account, but it won't work. It gives me the ID of 7e487h47m. What is going on?

**Comment by Jademalo** [comment]
![image](https://user-images.githubusercontent.com/386846/79702645-650cc200-829e-11ea-9480-0c6cd9edd236.png)

I'm getting this error too

**Comment by VoxeJ** [comment]
Me too

**Comment by Henriquelay** [comment]
Still getting this.

After a few minutes, found something about it on Reddit from u/gracenotes, sub mod and mantainer.
https://www.reddit.com/r/thewindmill/comments/gr4l4r/issue_while_logging_in_problem/ft0vf3f/
> The main reason for this issue is that the API that allowed sign-ins to happen was [shut down by Google](https://developers.google.com/%2B/api-shutdown) and stopped fully working around January 2020. Updating to a new version of signin requires updating everything, and I've already put over in a week of work to doing this, but still everything doesn't compile and link together, which has been demotivating.
> 
> It is at least still possible to make puzzles and share links to them without an account, since you can link to the editor page with the puzzle embedded in the URL.

**Comment by ErenBYilmaz** [comment]
I just recently got this error. But I see that some people are able to publish puzzles, one was published just a few days ago. Is there a way to publish without being signed in? Or is there some kind of workaround?

**Comment by Jademalo** [comment]
I have a feeling it was for people signed in before it broke, who still have existing auth cookies.

Two years later and I still can't sign in, very much feels abandoned unfortumately.

**Comment by Henriquelay** [comment]
Yes. It's not even doing PRs, one of us would have to fork and host an updated version. Problem is all those wonderful puzzles already registered in production would vanish. Does anyone have interest in doing so?
